.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: groovy
+jdk:
+- oraclejdk8
```

Comment:
> Travis CI is using 1.8.0_65 as is... that doesn't seem so bad. In the interest of saving build time, it seems best not do sudo apt-get update && sudo apt-get install oracle-java8-installer as your junit example does. What do you think?

If Travis CI is now using JDK 8 Update 65 as the default, then I agree: that should be fine.

For the JUnit 5 build, that wasn't the case when we set up the build: Travis CI used to default to an early JDK 8 release that didn't work for us.